makefile

```diff
@@ -137,3 +137,13 @@ setup_test_db:
 	createdb $$PGDATABASE; \
 	nz-buildings-load nz-buildings-plugin-db --with-plugin-setup; \
 	sed -i '4s/.*/dbname=nz-buildings-plugin-db/' ~/.qgis2/$(PLUGINNAME)/pg_config.ini
+
+check_ui_headers:
+	@echo
+	@echo "------------------------------------------"
+	@echo "Fix/revert header lines in .ui files to qgis.gui"
+	@echo "------------------------------------------"
+	for f in ./buildings/gui/*.ui; do \
+		sed -i.bak -e 's|<header>.*</header>|<header>qgis.gui</header>|g' $$f; \
```

Comment:
I think remove the `.bak` from this line then you don't need the next line at all - no need to create a backup of the file pre-changes if it's just going to be automatically deleted.